src/lib.rs

```diff
@@ -3,9 +3,13 @@
 #![cfg_attr(docsrs, feature(doc_cfg, external_doc))]
 #![deny(unsafe_code, missing_docs)]
 
-pub use self::address::{Address, Disconnected, WeakAddress};
+use std::error::Error as StdError;
```

Comment:
Not sure if that import is worth anything over a full-qualified use at the usage site.